src/main/java/com/philips/research/spdxbuilder/persistence/ort/OrtJson.java

```diff
@@ -131,15 +131,18 @@ public void updateProjectPackages(Map<String, URI> projectPackages) {
     @NullOr VcsJson vcsProcessed;
 
     Package createPackage() {
-        final var result = new Package(idElement(0).toLowerCase(), idElement(1), idElement(2), idElement(3));
-
+        @NullOr PackageURL packageUrl = null;
         if (purl != null) {
             try {
-                result.setPurl(new PackageURL(purl.toASCIIString()));
+                packageUrl = new PackageURL(purl.toASCIIString());
             } catch (MalformedPackageURLException e) {
-                throw new IllegalArgumentException(e);
+                // Ignore invalid PURL
```

Comment:
Is it safe to ignore this?

Reply:
The exception is thrown if the provided string is not actually a valid Package URL.

If this ever happens, I thought it would be quite safe to just act as if no Package URL was provided. But now you mention it, I am going to throw an exception so the user has to explicitly deal with this condition.